README.md

```diff
@@ -484,14 +484,6 @@ The `SafeDITool` utility is designed to able to be integrated into projects of a
 
 The `SafeDITool` can parse all of your Swift files at once, or for even better performance, the tool can be run on each dependent module as part of the build. Running this tool on each dependent module is currently left as an exercise to the reader.
 
-### Requirements
-
-* Xcode 16.0 or later
-* iOS 13 or later
-* tvOS 13 or later
-* watchOS 6 or later
-* macOS 10.13 or later
```

Comment:
deferring instead to `Package.swift`. Requirements are effectively "you can compile the Swift 6 toolchain and are compiling on a target that supports SwiftSyntax"